### PR TITLE
Algebraic connectivity

### PR DESCRIPTION
**Pull request by coryglover**
I have no clue how this past earlier tests.

### CODE REVIEW

File: netrw/rewire/algebraic_connectivity.py
Comment (nwlandry) [inline netrw/rewire/algebraic_connectivity.py:35]: This should have `copy_graph` as False by default.

Comment (coryglover) [reply netrw/rewire/algebraic_connectivity.py]: done. thanks for the catch

**Comment by samwestby** [comment]
I'm blocked on https://github.com/netsiphd/netrw/pull/14 until this is merged @sdmccabe @jkbren

**Review by sdmccabe — APPROVED** [review]

**Pull request merged**
